apis/overview/authorization.md

```diff
@@ -143,7 +143,7 @@ Native and single-page applications are public clients without a dedicated backe
 
 ### Authorization Code Flow with Proof Key for Code Exchange (PKCE)
 
-In order to mitigate these issue, OAuth 2.0 provides an option for Proof Key for Code Exchange (PKCE) by OAuth Public Clients (see OAuth 2.0 [RFC 7636](https://tools.ietf.org/html/rfc7636)). PKCE allows the calling application to dynamically generate a random, one -time key called a &quot;code verifier&quot;. Additionally, the calling app creates a transform of the &quot;code verifier&quot; called the &quot;code challenge&quot; and sends it to the authorization server when obtaining an authorization code. The authorization code obtained is then sent to the token endpoint with the &quot;code verifier &quot;and the server compares it with the previously received request code so that it can perform the proof of the &quot;code verifier&quot; by the client application. This provides a mitigation as the &quot;code verifier&quot; would be unknown to the attacker and cannot be intercepted as it is sent over TLS.
+In order to mitigate issues outlined in [previous section](#authorize-single-page-applications-spa-and-desktopmobile-applications-native), OAuth 2.0 provides an option for Proof Key for Code Exchange (PKCE) by OAuth Public Clients (see OAuth 2.0 [RFC 7636](https://tools.ietf.org/html/rfc7636)). PKCE allows the calling application to dynamically generate a random, one -time key called a &quot;code verifier&quot;. Additionally, the calling app creates a transform of the &quot;code verifier&quot; called the &quot;code challenge&quot; and sends it to the authorization server when obtaining an authorization code. The authorization code obtained is then sent to the token endpoint with the &quot;code verifier &quot;and the server compares it with the previously received request code so that it can perform the proof of the &quot;code verifier&quot; by the client application. This provides a mitigation as the &quot;code verifier&quot; would be unknown to the attacker and cannot be intercepted as it is sent over TLS.
```

Comment:
Have you checked whether link is correct? Seems like it is not working on portal.

Reply:
updated, but was not able to build the devPortal locally